c10/util/SmallBuffer.h

```diff
@@ -20,8 +20,8 @@ class SmallBuffer {
       "SmallBuffer is intended for POD types");
 
   T storage_[N];
```

Comment:
One minor nit while we are touching this file, can change this to an std::array?